docs/src/main/sphinx/functions/string.rst

```diff
@@ -134,6 +134,21 @@ String functions
 
     Removes trailing whitespace from ``string``.
 
+.. function:: soundex(char) -> string
+
+   ``soundex`` returns a character string containing the phonetic representation of ``char``.
+    This function is typically used to evaluate the similarity of two expressions phonetically, 
```

Comment:
It is typically used to evaluate the similarity of two expressions phonetically,  that is how the string sounds when spoken::

Reply:
Fixed it, apologies for this. Will do better next time.